### PR TITLE
Insomnia: Create API spec page

### PR DESCRIPTION
**Pull request by cloudjumpercat**
- I experimented with screenshots in this one. I noticed as we were doing group testing that it was difficult for testers to understand where they needed to go both when reading instructions and when being told verbally where to go. I'd like feedback on if these are useful or not.
- I realized we had an import API spec page but not create. Do people actually write a spec in Insomnia? Or are they always importing them?
- [The GraphQL info here](https://docs.insomnia.rest/insomnia/graphql-for-openapi) needed a place to live so I added it as a FAQ to the api spec pages. Does this work?

### CODE REVIEW

File: app/insomnia/api-specs.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+| Import an existing spec (requirements) | In your workspace, click **Import** and select the source of the import: **File**, **URL**, or **Clipboard**. Click **Scan**. You can review the resources to import, then click **Import**. |
+
+### Linting
+
+| Automatic linting on your specs (new or imported) directly in Insomnia | Navigate to the design document with your API spec. The linting error messages display below the center pane. ![Linting error messages](/assets/images/insomnia/linting-messages.png) |
+| Automate linting on specs using the Inso CLI | Run `inso lint spec` and select the spec from the list that displays. |
+| Use custom linting to ensure your spec adheres to your team or company's standards | To lint locally, create a `.spectral.yaml` file with the custom rules in the same directory as the `oas.yaml` file you want to lint. Then run `inso lint spec ./oas.yaml` from that directory. To apply custom lint rules in the Insomnia UI, add the `.spectral.yaml` file to the root of the collection git repository at the same level as the `.insomnia` folder. |
```
Comment (cloudjumpercat) [inline app/insomnia/api-specs.md:76]: Wasn't able to get this one to work locally, haven't tried git sync part yet (was hoping to follow Lucie's instructions once she opens that PR)

File: app/insomnia/api-specs.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+### Work with requests
+
+| Automatically generate a collection and requests from your spec | From your design document with your API spec, click the **Settings** icon and select **Generate collection**. ![Generate a collection](/assets/images/insomnia/generate-collection.png) |
+| Use mock servers to mock your requests | unsure how to work this properly |
```
Comment (cloudjumpercat) [inline app/insomnia/api-specs.md:81]: I tried to get this to work, but got stuck. the Mocks tab on a request just seems to send me to mock server and doesn't do anything? I think something must be up with the configuration.

File: app/insomnia/api-specs.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+Insomnia parses the following from your API specs and displays these in the Insomnia UI:
+
+| API spec definition | Object in Insomnia |
+| ---- | ---- |
+| `description` | Displays in the Docs tab of a request. |
+| `info` | Displays in the left-sidebar of the spec under INFO. |
+| `servers.url` | Displays in the left-sidebar of the spec under SERVERS. |
+| `paths` | Displays in the left-sidebar of the spec under PATHS. |
+| `requestBodies` | Displays in the left-sidebar of the spec under REQUEST BODIES. |
+| `schemas` | Displays in the left-sidebar of the spec under SCHEMAS. |
```
[the remark this answers is not in the document]
Comment (cloudjumpercat) [reply app/insomnia/api-specs.md:61]: Thanks! I added those now

File: app/insomnia/api-specs.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+  - text: Create a design document in Insomnia
+    url: /how-to/create-a-design-document/
+  - text: Generate a collection from a design document
+    url: /how-to/generate-a-collection-from-a-design-document/
+  - text: Lint API specs
+    url: /
+  - text: Automate linting
+    url: /
+  - text: Create a request
+    url: /how-to/create-a-request/
```
Comment (cloudjumpercat) [inline app/insomnia/api-specs.md:19]: @fabianrbz Oh, forgot to ask, I have this list of related resources here, but in the preview, the info box doesn't render with any information, including these. Any idea how I can fix this?

Comment (fabianrbz) [reply app/insomnia/api-specs.md:19]: @cloudjumpercat nice catch! I've pushed a fix to your branch.

**Pull request merged**